unfoldingWord/translationCore
# Change Folder Name From ULB

#### This pull request addresses:
This PR renames ULB file to originalLanguage


#### How to test this pull request:
Test project and see if Greek is there


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1795)
